frontend/src/main.ts

```diff
@@ -28,6 +28,16 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
 import { faTrashCan } from '@fortawesome/free-regular-svg-icons';
 import ErrorService from './services/ErrorService';
 
+import PrimeVue from 'primevue/config';
+
+// use bootstrap4 as default style
+import 'primevue/resources/themes/bootstrap4-light-blue/theme.css';
+import 'primevue/resources/primevue.min.css';
+import 'primeicons/primeicons.css';
+
+// import the component override style sheet for primevue
+import '@bcgov-nr/nr-fsa-theme/style-sheets/primevue-components-overrides.scss';
```

Comment:
Let's remove this one, cause we already imported it in the `asses/styles/styles/scss` file. But we need adjust the import order, put `import './assets/styles/styles.scss';` here. Cause if we import the `./assets/styles/styles.scss` file first, and then import the default primevue style, it will apply the default one on top of our custom stylesheet. The import order of the style file is important